### ISSUE TITLE
Remove double bylines

### ISSUE THREAD

**Issue by dorothyhernandez** [arsmentis/wdet-conversion#128]
There are two places where author(s) names are appearing. The one next to the date looks better but the one underneath the photo has more accurate because it shows all of the authors who contributed to a post.

<img width="564" alt="Screen Shot 2022-01-31 at 9 01 16 AM" src="https://user-images.githubusercontent.com/91985309/151807147-e6e4632e-ef25-46a0-b9eb-ee8ec564fcaf.png">

**Comment by dorothyhernandez** [comment]
@artlogic Is this something you could help me with? The name by the date appears to be the one who created the post in the CMS but the one under the photo is the correct information. My preference would be to have the authors' names currently appearing under the photo (because that makes it seem like it's a photo credit) where the name is currently by the date but if it's easier to just delete the name by the date that works too.

**Comment by dorothyhernandez** [comment]
The placement of the name looks good but depending on the post the author name/byline is not going to be correct.

For example, the content of this [post](https://wdet.org/2022/02/08/whitmer-to-present-spending-plan-this-week/) was authored by Rick Pluta, but his name doesn't appear at all on this post because mine, as the person who loaded the content into the CMS, is the one that appears.

The content in this [post](https://wdet.org/2022/02/07/just-kind-of-bracing-for-the-worst-possible-scenario-michigan-inmates-worry-as-omicron-spreads/) was authored by Michelle Jokisch Polo but her name is not appearing on the post because I am the user who loaded the content into the CMS and I changed it to admin so it doesn't look like I am taking credit for someone else's work.

**Comment by artlogic** [comment]
This will involve a careful reading of the theme source code and perhaps an override - it'll take time to actually resolve.

**Comment by artlogic** [comment]
@dorothyhernandez I've removed the incorrect byline in dev - can you review: https://wdetdev.wdet.org/2022/02/07/political-storylines-to-watch-this-year-in-michigan/

**Comment by dorothyhernandez** [comment]
@artlogic this is correct now but is it possible to have the author(s)' names by where the date is? The placement of the author(s)' names now is a bit unclear (like it looks like it's a photo credit since it's underneath the photo). Maybe if it said "By Dorothy Hernandez" instead of just "Dorothy Hernandez" that would help signal to the reader that it's a byline.

**Comment by Fw7424** [comment]
I reviewed this issue. I see on the development site, the top (native) byline has been removed. I'm assuming this is a configurable option within Wordpress and didn't require a code change. But I think the goal is to move the byline that comes from the plugin. 

Related files
covernews > inc > template-tags.php - built-in byline code is here
wdet35 > functions.php - secondary byline code - from plugin

Two possible options, though there may be others.
1. The data from the plugin might be able to be used to replace the built-in byline, via a code change.
2. Is there an existing plugin that would allow for this change.

**Comment by artlogic** [comment]
Waiting on PublishPress Authors Pro license info and CoverPress Pro 3.0 to hit prod.
